access-control-service-sql-impl/src/main/java/org/eclipse/tractusx/semantics/accesscontrol/sql/model/AccessRulePolicy.java

```diff
@@ -47,6 +47,7 @@ public class AccessRulePolicy {
    public static final String MANDATORY_SPECIFIC_ASSET_IDS_RULE_NAME = "mandatorySpecificAssetIds";
    public static final String VISIBLE_SPECIFIC_ASSET_ID_NAMES_RULE_NAME = "visibleSpecificAssetIdNames";
    public static final String VISIBLE_SEMANTIC_IDS_RULE_NAME = "visibleSemanticIds";
+   public static final String SPECIFIC_ASSET_ID_ANY_VALUE_IDENTIFIER = "any";
```

Comment:
Hi,
Should we worry about this special value being used as an actual value for a `specificAssetId` or is it impossible?

Reply:
done